content/en/citing-numpy.md

```diff
@@ -0,0 +1,28 @@
+**Citing NumPy**
```

Comment:
To create a page title, the format should be:
```
---
title: Citing NumPy
sidebar: false
---
```

Reply:
Thank you, @rgommers ! I'll keep it in mind next time.